srcpkgs/mongoose/template

```diff
@@ -1,29 +1,54 @@
 # Template file for 'mongoose'
 pkgname=mongoose
-version=6.18
-revision=4
+version=7.6
+revision=1
 build_style=gnu-makefile
-make_use_env=compliant
+make_use_env=yes
+make_check_target=test
+make_build_target=linux-libs
 makedepends="openssl-devel"
-short_desc="Easy to use Web server"
-maintainer="Orphaned <orphan@voidlinux.org>"
+checkdepends="openssl-devel"
+short_desc="Embedded Networking Library for TCP,UDP,HTTP,MQTT,WebSocket and SSL/TLS"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
 license="GPL-2.0-only"
-homepage="https://cesanta.com/"
+homepage="https://mongoose.ws/"
 distfiles="https://github.com/cesanta/mongoose/archive/${version}.tar.gz"
-checksum=f5c10346abc9c72f7cac7885d853ca064fb09aad57580433941a8fd7a3543769
+checksum=1ef09d971b6de1a6317c109980d6fb5a9c19b39efef2506d6b76869644b3dafa
+
+export SSL=OPENSSL
+CFLAGS+=" -fPIC"
```

Comment:
Appending to variables like `CFLAGS` at the top level is not a good idea because the template is sourced multiple times and the arguments can be repeated. Just set
```sh
CFLAGS="-fPIC"
```
instead. Better still, disable the `unused-function` error here and save the `pre_check` definition:
```sh
CFLAGS="-fPIC -Wno-error=unused-function"
```